sdks/java/core/src/main/java/org/apache/beam/sdk/coders/RowCoderGenerator.java

```diff
@@ -106,6 +106,7 @@
   private static final ByteBuddy BYTE_BUDDY = new ByteBuddy();
   private static final BitSetCoder NULL_LIST_CODER = BitSetCoder.of();
   private static final VarIntCoder VAR_INT_CODER = VarIntCoder.of();
+  private static final BitSet EMPTY_BIT_SET = new BitSet(0);
```

Comment:
I believe this is correct because BitSet.get(n) always returns false if n >= size. Please add a comment explaining this.